dev/build

```diff
@@ -57,6 +57,10 @@ Makefile.funcs.main() {
 	"$FUNC" "$@"	
 }
 
+# Ensure TMPDIR is set.
+: "${TMPDIR:="${RUNNER_TEMP:-}"}"
+[[ -n "$TMPDIR" ]] || die "Neither TMPDIR nor RUNNER_TEMP is set."
+
```

Comment:
This pattern is used all over the place. Locally `TMPDIR` is appropriate, unfortunately it's not set on GitHub runners, so we need to use their `RUNNER_TEMP` dir instead.